content/en/apps/tutorials/targets.md

```diff
@@ -16,53 +16,210 @@ This tutorial will take you through how to build target widgets.
+    appliesIf: function (contact, report) {
+      return Utils.getField(report, 'group_assessment.cough') === 'yes';
+    },
+    idType: 'contact',
+    date: 'reported'
+  }
+```
+
+### 4. Define the Cough Percentage Widget
+This widget calculates the percentage patients assessed that have been indicated to have a cough this month, regardless of the number of reports submitted for them.
+Edit the `targets.js` file and add the target widget as shown below:
+
+```javascript
+  {
+    id: 'percentage-contacts-with-cough-this-month',
+    type: 'percent',
+    icon: 'icon-healthcare-assessment',
+    goal: -1,
+    translation_key: 'targets.assessments.percentage.cough.title',
+    percentage_count_translation_key: 'targets.assessments.percentage.with.cough',
+    subtitle_translation_key: 'targets.this_month.subtitle',
+    appliesTo: 'reports',
+    appliesToType: ['assessment'],
+    appliesIf: function (contact) {
+      return contact.contact && contact.contact.parent && contact.contact.parent.parent && contact.contact.parent.parent.parent;
+    },
+    passesIf: function(contact, report) {
+      return Utils.getField(report, 'group_assessment.cough') === 'yes';
+    },
+    idType: 'contact',
+    date: 'reported'
+  }
+```
+
+### 5. Define Total Households with Assessments Widget
+This widget calculates the number of households that have patients assessed this month. Note that `emitCustom` emits a custom target instance object. Filter based on reports, and use the contact information from the reports to emit target instances. The target instance ID is the household ID.
+Edit the `targets.js` file and add the target widget as shown below:
+
+```javascript
+ {
+    id: 'households-with-assessments-this-month',
+    type: 'count',
+    icon: 'icon-healthcare-assessment',
+    goal: 4,
+    translation_key: 'targets.households.with.assessments.title',
+    subtitle_translation_key: 'targets.this_month.subtitle',
+    appliesTo: 'reports',
+    appliesToType: ['assessment'],
+    date: 'reported',
+    emitCustom: (emit, original, contact) => {
+      const householdId = contact.contact && contact.contact.parent._id;
+      emit(Object.assign({}, original, {
+        _id: householdId,
+        pass: true
+      }));
+    },
+    dhis: { //The IDs should correspond to the IDs on DHIS site
+      dataSet: 'VMuFODsyWaO',
+      dataElement: 'kB0ZBFisE0e'
+    }
+  },
+```
+
+### 6. Define Households with Assessments Widget
+This widget calculates the number of households that have two or more patients assessed this month. Use `emitCustom` to emit a custom target instance object. The target instance ID is the household ID.
+Edit the `targets.js` file and add the target widget as shown below:
+
+```javascript
+
+//Define a function to get the household ID
+const getHouseholdId = (contact) => contact.contact && contact.contact.type === 'clinic' ? contact.contact._id : contact.contact.parent && contact.contact.parent._id;
```

Comment:
You should probably call out that you are using the default contact types and link to the reference page talking about that and hierachies